lib/rucio/db/sqla/__init__.py

```diff
@@ -3,3 +3,27 @@
 # Licensed under the Apache License, Version 2.0 (the "License");
 # You may not use this file except in compliance with the License.
 # You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+from sqlalchemy.sql.expression import bindparam, text
+
+
+def filter_thread_work(session, query, total_threads, thread_id, hash_variable=None):
```

Comment:
Can you move this to `db.sqla.util` instead. I think we should avoid having methods in the `__init__`
Maybe we had this discussion already and you did this due to circular dependencies, in that case, please ignore my comment 😄